Library/CountBadgeView.swift

```diff
@@ -23,19 +23,19 @@ public final class CountBadgeView: UIView {
   public override func bindStyles() {
     let required = UILayoutPriorityRequired
 
-    self
+    _ = self
```

Comment:
well, our uikit lenses are inherently mutative and we discard the result of piping values through a transformation block. so we either have to do `_ = ` or find a way to add `@discardableResult` to some of our operators... :/